strawberry/schema/types/base_scalars.py

```diff
@@ -67,3 +67,17 @@ def parse_decimal(value: str) -> decimal.Decimal:
     serialize=str,
     parse_value=wrap_parser(uuid.UUID, "UUID"),
 )
+
+
+def _verify_void(x) -> None:
+    if x is not None:
+        raise ValueError(f"Expected 'None', got '{x}'")
+
+
+Void = scalar(
+    type(None),
+    name="Void",
+    serialize=_verify_void,
+    parse_value=_verify_void,
+    description="Represents NULL values",
```

Comment:
```suggestion
    description="Represents None values",
```
maybe it is better to use None here? or Void? since it will be used in the schema :)

Reply:
I think `NULL` makes sense as it will be exposed on on the GraphQL protocol, while `None` is a Python thing. 

However I don't have strong feeling about that :man_shrugging: 

For whatever it is worth, I have defined this description to be the consistent with [graphql-scalar's Void](https://github.com/Urigo/graphql-scalars/blob/master/src/scalars/Void.ts#L7)